codegens/php-httprequest2/test/newman/newman.test.js

```diff
@@ -0,0 +1,19 @@
+var runNewmanTest = require('../../../../test/codegen/newman/newmanTestUtil').runNewmanTest,
+  convert = require('../../index').convert;
+
+
+describe('PHP HttpRequest2 Converter', function () {
+  // Skipping newman tests for now
+  // TODO: Addition of set_include_path() in generated snippet
+  describe.skip('convert for different request types', function () {
```

Comment:
We can remove this file and npm/test-newman.js for now if we are not planning to add newman tests right now.

Reply:
We can keep it for future reference, as I have added a TODO to note what is left to be added.